### PR TITLE
Code standards updated to PHP 7.1

### PR DESCRIPTION
**Pull request by TavoNiievez**
- The minimum version of PHP is now 7.1
- Added strict types
- Added return types
- Added typehints
- Removed unnecessary qualifiers

### CODE REVIEW

File: src/Codeception/Module/SOAP.php [hunk 1]
```diff
@@ -105,34 +124,34 @@ protected function onReconfigure()
 
     public function _depends()
     {
-        return ['Codeception\Lib\InnerBrowser' => $this->dependencyMessage];
+        return [\Codeception\Lib\InnerBrowser::class => $this->dependencyMessage];
```
Comment (Naktibalda) [inline src/Codeception/Module/SOAP.php]: Is there a reason to use fully qualified name here?

File: src/Codeception/Module/SOAP.php [hunk 2]
```diff
@@ -105,34 +124,34 @@ protected function onReconfigure()
...
+    private function getXmlResponse(): DOMDocument
     {
         if (!$this->xmlResponse) {
             throw new ModuleException($this, "No XML response, use `\$I->sendSoapRequest` to receive it");
         }
         return $this->xmlResponse;
     }
 
-    private function getXmlStructure()
+    private function getXmlStructure(): \Codeception\Util\XmlStructure
```
Comment (Naktibalda) [inline src/Codeception/Module/SOAP.php]: FQDN here too

File: src/Codeception/Module/SOAP.php [hunk 3]
```diff
@@ -164,16 +183,15 @@ private function getXmlStructure()
      * ```
      *
      * @param $header
-     * @param array $params
      */
-    public function haveSoapHeader($header, $params = [])
+    public function haveSoapHeader($header, array $params = []): void
```
Comment (Naktibalda) [inline src/Codeception/Module/SOAP.php]: An absolute majority of untyped parameters in this class are strings.

**Review by Naktibalda — CHANGES_REQUESTED** [review]

File: src/Codeception/Module/SOAP.php (hunk 2 above)
Comment (TavoNiievez) [reply src/Codeception/Module/SOAP.php]: In fact, I imagined you telling me that if it were simply called 'XmlStructure', the information that it belongs to \Codeception would be lost when reading this line.

But you're right. I'll change it.

**Review by Naktibalda — APPROVED** [review]

**Pull request merged**